tests/src/InputCompletion.cpp

```diff
@@ -222,15 +226,25 @@ class InputCompletionTest : public ::testing::Test
 
 TEST_F(InputCompletionTest, EmoteNameFiltering)
 {
+    // The completion doesn't guarantee an ordering,
+    // execpt for exact matches.
+    const auto cmpCompletion = [](const auto &a, const auto &b) {
+        return a.displayName < b.displayName;
+    };
+
     auto completion = queryEmoteCompletion(":feels");
     ASSERT_EQ(completion.size(), 3);
-    ASSERT_EQ(completion[0].displayName, "FeelsBirthdayMan");
-    ASSERT_EQ(completion[1].displayName, "FeelsBadMan");
+    // all these matches are BTTV global emotes
+    std::sort(completion.begin(), completion.end(), cmpCompletion);
```

Comment:
I'm hesitant about this change - I'd rather leave this change out as part of the platform-agnostic PR & discuss it in a separate PR/issue.
Most likely I'd like to see us just change the EmoteMap to be consistently ordered instead

Reply:
I reverted the change, but added a note for future visitors.

> Most likely I'd like to see us just change the EmoteMap to be consistently ordered instead

Not sure if it's that easy. It should probably be benchmarked to see if it makes a meaningful difference, since working with emotes and checking for emotes in a word is an integral part.